packages/ember-htmlbars/tests/helpers/view_test.js

```diff
@@ -366,436 +61,99 @@ QUnit.test('Should apply class without condition always', function() {
-  var LabelView = EmberView.extend({
-    classNameBindings: ['something'],
-    something:         'visible'
-  });
-
-  owner.register('controller:label', Controller, { instantiate: true });
-  owner.register('view:label', LabelView);
-  owner.register('template:label', compile('<div id="child-view"></div>'));
-  owner.register('template:nester', compile('{{render "label"}}'));
-
-  view = EmberView.create({
-    [OWNER]: owner,
-    templateName: 'nester'
-  });
-
-  runAppend(view);
-
-  ok(view.$('.visible').length > 0, 'class bindings are not overriden');
-});
-
-QUnit.test('child views can be inserted using the {{view}} helper', function() {
-  owner.register('template:nester', compile('<h1 id="hello-world">Hello {{world}}</h1>{{view view.labelView}}'));
-  owner.register('template:nested', compile('<div id="child-view">Goodbye {{cruel}} {{world}}</div>'));
-
-  var context = {
-    world: 'world!'
-  };
-
-  var LabelView = EmberView.extend({
-    tagName: 'aside',
-    templateName: 'nested'
-  });
-
-  view = EmberView.create({
-    [OWNER]: owner,
-    labelView: LabelView,
-    templateName: 'nester',
-    context: context
-  });
-
-  set(context, 'cruel', 'cruel');
-
-  runAppend(view);
-
-  ok(view.$('#hello-world:contains("Hello world!")').length, 'The parent view renders its contents');
-  ok(view.$('#child-view:contains("Goodbye cruel world!")').length === 1, 'The child view renders its content once');
-  ok(view.$().text().match(/Hello world!.*Goodbye cruel world\!/), 'parent view should appear before the child view');
-});
-
-QUnit.test('should be able to explicitly set a view\'s context', function() {
-  var context = EmberObject.create({
-    test: 'test'
-  });
-
-  var CustomContextView = EmberView.extend({
-    context: context,
-    template: compile('{{test}}')
-  });
-
-  view = EmberView.create({
-    customContextView: CustomContextView,
-    template: compile('{{view view.customContextView}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'test');
-});
-
-QUnit.test('Template views add an elementId to child views created using the view helper', function() {
-  owner.register('template:parent', compile('<div>{{view view.childView}}</div>'));
-  owner.register('template:child', compile('I can\'t believe it\'s not butter.'));
-
-  var ChildView = EmberView.extend({
-    templateName: 'child'
-  });
-
-  view = EmberView.create({
-    [OWNER]: owner,
-    childView: ChildView,
-    templateName: 'parent'
-  });
-
-  runAppend(view);
-
-  var childView = get(view, 'childViews.firstObject');
-  equal(view.$().children().first().children().first().attr('id'), get(childView, 'elementId'));
-});
-
-QUnit.test('Child views created using the view helper should have their parent view set properly', function() {
-  var template = '{{#view}}{{#view}}{{view}}{{/view}}{{/view}}';
-
+QUnit.test('Should not apply classes when bound property specified is false', function() {
```

Comment:
Now covered in https://github.com/emberjs/ember.js/pull/13236/files#diff-c47395a5aaf92aa0a7edbfa812d4e0afR108